modules/rhcos-enabling-multipath.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="rhcos-enabling-multipath_{context}"]
+= Enabling multipathing with kernel arguments on {op-system}
+
+{op-system} supports multipathing on the primary disk, allowing stronger resilience to hardware failure so that you can set up {op-system} on top of multipath to achieve higher host availability.
```

Comment:
This sentence looks like it's repeating itself. Maybe just:

> {op-system} supports multipathing on the primary disk, allowing stronger resilience to hardware failure to achieve higher host availability.

?

Reply:
Resolved